main.cpp

```diff
@@ -53,83 +53,83 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 //  1)
 //  Noun:        plant
-//  action 1:    bear
-plant.bear();
-//  action 2:    bloom
+//  action 1:    the plant bears fruits
+plant.bearFruits();
+//  action 2:    the plant blooms
 plant.bloom();
-//  action 3:    seed
-plant.seed();
+//  action 3:    the plant produces seeds
+plant.produceSeeds();
 //  2)
 //  Noun:        cat
-//  action 1:    meowing
-cat.meowing();
-//  action 2:    purr
+//  action 1:    the cat meows
+cat.moew();
+//  action 2:    the cat purrs
 cat.purr();
-//  action 3:    lapping
-cat.lapping();
+//  action 3:    the cat drinks milk
+cat.drinkMilk();
 //  3)
 //  Noun:        wind
-//  action 1:    blowing
-wind.blowing();
-//  action 2:    gusting
-wind.gusting();
-//  action 3:    whooshing
-wind.whooshing();
+//  action 1:    the wind blows
+wind.blow();
+//  action 2:    the wind howls
+wind.howl();
+//  action 3:    the wind dies down
+wind.dieDown();
 //  4)
 //  Noun:        people
-//  action 1:    screaming
-people.screaming();
-//  action 2:    cheering
-people.cheering();
-//  action 3:    crying
-people.crying();
+//  action 1:    people scream
+people.scream();
+//  action 2:    people cheer
+people.cheer();
+//  action 3:    people cry
+people.cry();
 //  5)
 //  Noun:        bird
-//  action 1:    chirping
-bird.chirping();
-//  action 2:    tweeting
-bird.tweeting();
-//  action 3:    fly
-bird.flying();
+//  action 1:    the bird sings
+bird.sing();
+//  action 2:    the bird builds a nest
+bird.buildNest();
+//  action 3:    the bird migrates
+bird.migrate();
 //  6)
 //  Noun:        energy
-//  action 1:    boost
+//  action 1:    the energy boosts
 energy.boost();
-//  action 2:    propagate
+//  action 2:    the energy propagates
 energy.propagate();
-//  action 3:    pulsate
+//  action 3:    the energy pulsates
 energy.pulsate();
 //  7)
 //  Noun:        water
-//  action 1:    flow
+//  action 1:    the water flows
 water.flow();
-//  action 2:    boil
+//  action 2:    the water boils
 water.boil();
-//  action 3:    drip
-water.drip();
+//  action 3:    the water absorbs oxygen
+water.absorbOxygen();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.